docs/docs/10.4-test-utils.md

```diff
@@ -6,15 +6,15 @@ prev: two-way-binding-helpers.html
 next: clone-with-props.html
 ---
 
-`ReactTestUtils` makes it easy to test React components in the testing framework of your choice (we use [Jest](https://facebook.github.io/jest/)).
+`ReactTestUtils` makes it easy to test React components in the testing framework of your choice. At Facebook we use [Jest](https://facebook.github.io/jest/) for painless JavaScript testing. Learn how to get started with Jest through the Jest website's [React Tutorial](http://facebook.github.io/jest/docs/tutorial-react.html#content).
 
 ```
 var ReactTestUtils = require('react-addons-test-utils');
 ```
 
 > Note:
 > 
-> Airbnb has released a testing utility called Enzyme, which makes it easy to assert, manipulate, and traverse your React Components' output. If you're deciding on a unit testing library, it's worth checking out: [http://airbnb.io/enzyme/](http://airbnb.io/enzyme/)
+> Airbnb has released a testing utility called Enzyme, which makes it easy to assert, manipulate, and traverse your React Components' output. If you're deciding on a unit testing utility to use together with Jest, it's worth checking out: [http://airbnb.io/enzyme/](http://airbnb.io/enzyme/)
```

Comment:
I'm not sure, maybe add something to make it clear that it enzyme can be used with other testing libraries. Something like:

> If you're deciding on a unit testing utility to use together with Jest, or another testing library, it's worth checking out